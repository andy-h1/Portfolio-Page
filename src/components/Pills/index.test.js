import React from "react";
import { render } from "@testing-library/react";
import { Pills } from ".";

const props = {
  items: ["react", "styled-components", "jest", "react-testing-library"],
};

describe("Pills Component", () => {
  it("should render", () => {
    const { container } = render(<Pills {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
