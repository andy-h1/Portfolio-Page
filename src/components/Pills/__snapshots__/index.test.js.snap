// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pills Component should render 1`] = `
<ul
  class="sc-AxjAm iQwMNU"
>
  <li
    class="sc-AxirZ ikOfEZ"
  >
    react
  </li>
  <li
    class="sc-AxirZ ikOfEZ"
  >
    styled-components
  </li>
  <li
    class="sc-AxirZ ikOfEZ"
  >
    jest
  </li>
  <li
    class="sc-AxirZ ikOfEZ"
  >
    react-testing-library
  </li>
</ul>
`;
